x/deployment/client/cli/tx.go

```diff
@@ -32,10 +40,154 @@ func GetTxCmd(key string, cdc *codec.Codec) *cobra.Command {
 		cmdUpdate(key, cdc),
 		cmdClose(key, cdc),
 		cmdGroupClose(key, cdc),
+		makeItSoCommand(key, cdc),
 	)...)
 	return cmd
 }
 
+func makeItSoCommand(key string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "make-it-so [sdl-file]",
+		Short: fmt.Sprintf("Create %s", key),
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Setup CLI context and transaction builder
+			ctx := context.NewCLIContext().WithCodec(cdc)
+			bldr := auth.NewTxBuilderFromCLI(os.Stdin).WithTxEncoder(utils.GetTxEncoder(cdc))
+			if err := ctx.Client.Start(); err != nil {
+				return err
+			}
+
+			// Read in manifest file
+			sdl, err := sdl.ReadFile(args[0])
+			if err != nil {
+				return err
+			}
+
+			// Fetch deployment groups from the SDL file, each group maps to an individual order
+			groups, err := sdl.DeploymentGroups()
+			if err != nil {
+				return err
+			}
+
+			// Fetch the manifest from the SDL file
+			mani, err := sdl.Manifest()
+			if err != nil {
+				return err
+			}
+
+			// Get the deployment ID (defaults to height)
+			id, err := DeploymentIDFromFlags(cmd.Flags(), ctx.GetFromAddress().String())
+			if err != nil {
+				return err
+			}
+
+			// Default DSeq to the current block height
+			if id.DSeq == 0 {
+				if id.DSeq, err = currentBlockHeight(ctx); err != nil {
+					return err
+				}
+			}
```

Comment:
We should wrap all the Msg creation including the 
```
			msg := types.MsgCreateDeployment{
				ID: id,
				// TODO: Version:  []byte{0x1, 0x2},
				Groups: make([]types.GroupSpec, 0, len(groups)),
			}
```
 into a utility function. It's also called in normal Create, and it'd be good not to have drift.

Reply:
Agree, I'll add this.